Nos/Service/CurrentUser.swift

```diff
@@ -38,7 +38,9 @@ enum CurrentUser {
     }
     
     static var author: Author {
-        try! Author.findOrCreate(by: publicKey!, context: context!)
+        let persistenceController = PersistenceController.shared
+        context = persistenceController.container.viewContext
+        return try! Author.findOrCreate(by: publicKey!, context: context!)
```

Comment:
I tried removing my key from Settings and relaunching the app and it crashes on this line. Maybe we should make this optional?

Reply:
Ah, good catch. Yes, I'l fix that.